frontsite/views.py

```diff
@@ -19,3 +23,35 @@ def index(request):
             'form': form,
             'job': job
         })
+
+def user_login(request):
+    template = 'login.html'
+
+    if request.method == 'POST':
+        # Gather the username and password provided by the user.
```

Comment:
Please remove redundant comments. Re-read your written code, comment lines that you find out it's difficult to understand as a code reader. For example, `username = request.POST.get('username')`, this line has a clear meaning that you want to get username key from POST dict => comment is redundant here.

Reply:
The comment is not about what that line of code does. It's a note about the reason we're choosing request.POST.get() over request.POST. Thus I found it necessary.

Reply:
`request.POST` is a dict in Python. The purpose of this class's `get` method is helping developer to control returned value better. The note is already there when you type `.get(<key>)` after `request.POST`, which makes I find these comments redundant.